app/views/admin/registrations/index.html.haml

```diff
@@ -25,6 +25,9 @@
             %th ID#
             %th Name
             %th E-Mail
+            - if @code_of_conduct
```

Comment:
I think you should use `.present?` as we don't want to show it if is empty. Do we? :thinking:

Reply:
`@code_of_conduct = @conference.code_of_conduct.present?`